packages/cli/src/cmds/agentInstaller/agentProcedure.ts

```diff
@@ -251,7 +252,14 @@ export default abstract class AgentProcedure {
         UI.error(addGitReturn.errorMessage);
       }
 
-      const commitGitReturn = await this.gitCommit(filesDiff, 'feat: Install AppMap.');
+      const commitGitReturn = await this.gitCommit(filesDiff, `feat: Configure AppMap for this project
+
+AppMap is a free and open-source runtime code analysis tool.
+
+AppMap has been installed and configured per installation instructions
```

Comment:
Maybe slightly clearer would be "...and configured using the installation instructions at:"

Reply:
Reworded.